src/main/java/com/titanic/airbnbclone/repository/AccommodationRepository.java

```diff
@@ -59,16 +61,15 @@
         String queryString = "select distinct a from Accommodation as a left join fetch a.pictures";
         List<Accommodation> accommodations = entityManager
                 .createQuery(queryString, Accommodation.class)
-                .setFirstResult(0)
-                .setMaxResults(30)
                 .getResultList();
+        accommodations = accommodations.subList(0, 30);
```

Comment:
0, 30 등과 같은 매직넘버(숫자로 직접 변수)주는건 가능하면 따로 빼는건 어떨까요?
```suggestion
        accommodations = accommodations.subList(START, ACCOMODATIONCOUNT);
```

Reply:
네네 매직넘버는 지양해야죠 
와 ㅋㅋ 제안한 변화를 바로 반영할 수 있군요